doc/nrf/ug_bootloader_external_flash.rst

```diff
@@ -3,13 +3,66 @@
 Using external flash memory partitions
 ######################################
 
+
 When using MCUboot, you can store the storage partition for the secondary slot in the external flash memory, using a driver for the external flash memory that supports the following features:
 
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+.. note::
+
+   The Partition Manager will only allow run-time access to flash partitions that are
+   placed on external flash devices that have drivers compiled in. Partition Manager has no
```

Comment:
Instead of mentioning partitions placed on external flash devices, would it be more precise to talk about "regions" located in external flash? Any partition needs to be stored in a region, and a region can be located in external flash. Just a thought.